wordpress-mobile/WordPress-Android
# Show BottomSheet with Gravatar info before continuing with avatar change

Fixes #20190

We want the user to notice that if they modify their avatar in the app, they are modifying their avatar in Gravatar. For that reason, we'll show a bottom sheet with Gravatar information.

| Light | Dark |
|----------|-----------|
|<img width="250" alt="Alt Text" src="https://github.com/wordpress-mobile/WordPress-Android/assets/6974554/b981e54b-414f-4639-a61c-103c8102319f">|<img width="250" alt="Alt Text" src="https://github.com/wordpress-mobile/WordPress-Android/assets/6974554/1783e20a-1704-4d80-a747-db6d1e0c4fcb">|

| Big font size | Landscape / scrollable |
|----------|-----------|
|<img width="250" alt="Alt Text" src="https://github.com/wordpress-mobile/WordPress-Android/assets/6974554/28186cb4-79e2-40a1-a6c4-1970e20f832f">|<img width="250" alt="Alt Text" src="https://github.com/wordpress-mobile/WordPress-Android/assets/6974554/81cdc9ae-f7d5-4cf1-bfd9-19a24c401173">|

**Note:** In dark mode, the navigation bar colors can be improved. I've tried, but I haven't found an easy way to do this. I don't think it should be blocking, but let me know what you think (or if you have any suggestions). We can try to iterate over this in following PRs.

## To Test:

Try on both JP and WP:

1. Go to the profile tab
2. Click on the `Avatar`
3. Observe the BottomSheet with Gravatar information is opened
4. Click on `Learn more`
5. Observe Gravatar's Url is open in an external browser
6. Go back to the app
7. Click `Continue`
8. Observer the image selector is opened
9. Go back
10. Click on the `Avatar` again
11. Close the BottomSheet (pressing back or dragging)
12. Observe nothing happens

-----

## Regression Notes

1. Potential unintended areas of impact

    - We shouldn't impact other areas rather than the avatar update flow. We are adding a BottomSheet in  MeFragment, so all possible impacts should be inside that fragment.

13. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

14. What automated tests I added (or what prevented me from doing so)

    - N/A

-----

## PR Submission Checklist:


-----

## Testing Checklist:
